mithril-core/src/stm.rs

```diff
@@ -378,16 +393,14 @@ impl<D: Clone + Digest> StmSigner<D> {
             }
         }
         if !indexes.is_empty() {
-            let path = self
-                .closed_reg
-                .merkle_tree
-                .get_path(self.mt_index.try_into().ok()?);
             Some(StmSig {
                 sigma,
                 pk: self.vk,
                 stake: self.stake,
                 indexes,
-                path,
+                // path: Path::default(),
+                signer_index: Option::from(self.mt_index),
```

Comment:
Correct notation is 
```suggestion
                signer_index: Some(self.mt_index),
```

but I still don't see why we need an option.